src/Web/Avalonia.Web.Blazor/wwwroot/Avalonia/Avalonia.js

```diff
@@ -0,0 +1,6 @@
+export { DpiWatcher } from "./DpiWatcher";
```

Comment:
I believe that's a duplicate file. And all other files in wwwroot too.

Reply:
Yes, duplicate